libethcore/EthashAux.cpp

```diff
@@ -87,13 +87,10 @@ ethash_params EthashAux::params(h256 const& _seedHash)
 {
 	Guard l(get()->x_epochs);
 	unsigned epoch = 0;
-	try
+	auto epochIter = get()->m_epochs.find(_seedHash);
+	if (epochIter == get()->m_epochs.end())
 	{
-		epoch = get()->m_epochs.at(_seedHash);
```

Comment:
odd - there should only be at most the number of epochs in the current chain i.e. < 10.